go.mod

```diff
@@ -1,13 +1,13 @@
-module github.com/open-cluster-management/cert-policy-controller
+module github.com/stolostron/cert-policy-controller
 
 go 1.16
 
 require (
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.13.0
 	github.com/open-cluster-management/addon-framework v0.0.0-20210621074027-a81f712c10c2
```

Comment:
/lgtm

Could you open a new issue to track all sibling squads dependence renaming in the future?

One issue for all grc repos should be fine.

Reply:
Good idea--done!
- https://github.com/stolostron/backlog/issues/18820